integration/config/configtx.yaml

```diff
@@ -36,19 +44,22 @@ Organizations:
         Policies: &SampleOrgPolicies
             Readers:
                 Type: Signature
-                Rule: "OR('SampleOrg.member')"
```

Comment:
Why did you change this from the sampleconfig version?  I think they also are functionally identical?  You probably added it in the attempt of getting raft working.  I think we should that though only cleanly in a separate PR with this PR cleanly based on fabric sampleconfig, a half-way conversion to raft mixed with a v1 to v2 conversion . This also applies to many related changes below